### ISSUE TITLE
PCDS ADStream Incompatibilities with Ophyd Areadetector

### ISSUE THREAD

**Issue by ZLLentz** [pcdshub/pcdsdevices#360]
## Expected Behavior

Area detector stuff works with ADStream

## Current Behavior

The PVs are different enough that it fails
TODO: fully explore differences

## Possible Solution

Make custom stuff...

**Comment by hhslepicka** [comment]
Shouldn't ADStream be fixed in the interest of being AreaDetector compatible?

**Comment by ZLLentz** [comment]
Yes, but the first step either way is to identify all the differences

**Comment by hhslepicka** [comment]
Makes sense
